src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -252,6 +252,7 @@ Object {
       },
     },
     "margin": Object {
+      "-1": "-1px",
```

Comment:
in the paddings, we use `negative` for the key for `-1px`. I'm open to any variation, but would welcome consistency

Reply:
can you please check if this value is in the shared config already?

Reply:
> in the paddings, we use `negative` for the key for `-1px`. I'm open to any variation, but would welcome consistency

I think we are using `negative` only for the z-index key and not the padding. I checked how it is done in dealerhub and listings and there it is `"-1": "-1px"`
Should I change them to use negative as the key?

Reply:
generally, I would push for as much consistency as possible. I'm wondering whether we need to define specific `-`/negative keys at all. I would assume tailwind generates them from the regular ones, for example `-mt-5` and `mt-5` when there's a `"5": "5px"`

Reply:
that said - it depends on your capacity. if you'd like to clean up, please do. if you feel it's a rabbit hole that's not worth it, try something sensible in the context you're in. it's nice to continually clean up on the side, but it should not prevent us from shipping stuff

tl;dr: as long as we don't make it harder to mantain, I'm fine with finding that there's something we can optimize

Reply:
I tested it, `-` is required on custom values. please see here for reference: https://tailwindcss.com/docs/margin#negative-values

But in general, I think the custom `-1` is not needed along with other custom values we defined in the config because tailwind already defines those. And I think as soon as we overwrite them, the functionality that you can use +/- is gone and you have to define both.

I can try to remove them but I'd prefer to create another ticket so that it gets tested well. Is this ok?

Reply:
thanks for confirming. I was off on this :)

can absolutely be a separate thing. just this one would be a tiny improvement only. would only tackle it if we accidentally duplicated some values with different keys. after all, this will generate global css, which turns out to be very hard to reduce

Reply:
Will check all of them as my next [task](https://autoricardo.atlassian.net/browse/CFYEO-40)